Formula/dmd.rb

```diff
@@ -3,22 +3,22 @@ class Dmd < Formula
   homepage "https://dlang.org/"
 
   stable do
```

Comment:
Remove the `stable do` block, which is unnecessary now.

Reply:
Note there's still a `head do` so it's actually still needed.